pkgs/tools/security/sherlock/default.nix

```diff
@@ -15,13 +16,13 @@ let
 in
 stdenv.mkDerivation rec {
   pname = "sherlock";
-  version = "0.14.0";
+  version = "0.14.3";
```

Comment:
Could you change to `0.14.1` ?
It seems it *was* `0.14.3` but it is that version.
https://github.com/sherlock-project/sherlock/compare/ab2f678..1ffcaa5#diff-30cfd77c3faaddd8cf5b5d2a4d8f3bbe9dc265ba5eebcc0cb2be3494ec02f992L31

Reply:
Using `sherlock --version` still returns `0.14.3` as of the last commit in that commit comparison link. 🤔

Reply:
Sorry, I am really confused by the upstream diff (the way github represents). but the commit updating to contains https://github.com/sherlock-project/sherlock/commit/d7fce940473848072e8deda541f02f403daa1e87 which is the last edit to sherlock.py and it's updating to version `0.14.3`. so the behabiour is correct. Approved!